app/views/sections/show.html.erb

```diff
@@ -23,6 +23,11 @@
     <li>
       <% if LessonSection.find_by(lesson: lesson, section: @section).homework? %>
         <i class="glyphicon glyphicon-book"></i>
+      <% else %>
+        <i class="glyphicon glyphicon-pencil"></i>
+      <% end %>
+      <% if LessonSection.find_by(lesson: lesson, section: @section).try(:day_of_week) %>
+        <%= LessonSection.find_by(lesson: lesson, section: @section).day_of_week.titlecase %>
```

Comment:
Yep!  Just checked and there is a default for both so no worries that
lessons that haven't been updated would be broken.  Just need to get to
updating...(until then everything will look like weekend homework...)

As far as presence validation, I don't know.  Theoretically, lessons could
exist that are not specific to a day or type but there aren't currently
any.  I will add that, too, with the other changes.

k

On Wed, Dec 30, 2015 at 1:12 PM, Michael Kaiser-Nyman <
notifications@github.com> wrote:

> In app/views/sections/show.html.erb
> <https://github.com/epicodus/textbook/pull/28#discussion_r48631792>:
>
> > @@ -23,6 +23,11 @@
> >      <li>
> >        <% if LessonSection.find_by(lesson: lesson, section: @section).homework? %>
> >          <i class="glyphicon glyphicon-book"></i>
> > +      <% else %>
> > +        <i class="glyphicon glyphicon-pencil"></i>
> > +      <% end %>
> > +      <% if LessonSection.find_by(lesson: lesson, section: @section).try(:day_of_week) %>
> > +        <%= LessonSection.find_by(lesson: lesson, section: @section).day_of_week.titlecase %>
>
> How about replace 29 and 30 with:
>
> <%= LessonSection.find_by(lesson: lesson, section: @section).try(:day_of_week).try(:titlecase) %>
>
> Also, shouldn't day of week and work type have a presence validation? If
> so, then we should just add that and remove the trys.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/epicodus/textbook/pull/28/files#r48631792>.
>